web/packages/teleterm/src/ui/ModalsHost/modals/ReAuthenticate/ReAuthenticate.tsx

```diff
@@ -162,7 +175,7 @@ export const ReAuthenticate: FC<{
                   {isLeafCluster && ` from trusted cluster "${clusterName}"`}
                 </Text>
 
-                <Flex width="100%" gap={3} flex-wrap="no-wrap">
```

Comment:
Heh, this didn't work, I must've made that mistake a long time ago. "no-wrap" is the default anyway. But if we set `flexWrap` to `wrap`, then we can show the TOTP input on the same line as the select. When an alert is supposed to be shown, we can show an empty box with `flex={1}` so that it occupies half the width of the parent that the TOTP input would otherwise occupy, and then we can add an alert with width set to 100% so that it wraps to the next row.